virtme/commands/run.py

```diff
@@ -968,6 +968,28 @@ def cleanup_console_script():
     qemuargs.extend(["-device", "vhost-vsock-pci,guest-cid=%d" % args.port])
 
 
+def ssh_client(args):
+    remote_cmd = args.remote_cmd if args.remote_cmd else ""
+    cmd = "ssh -p %d localhost %s" % (args.port, remote_cmd)
```

Comment:
for the remote command, would it not be better to execute it in a subshell, e.g. with `bash -c "..."`? By doing that, it will be possible to pass longer commands, e.g. separated by semicolons.

So we could have something similar to the console mode:

```python
exec_escaped = args.remote_cmd.replace('"', '\\"')
cmd = f'bash -c "{exec_escaped}"'
```